tests/dbz-twin/rac/db-check.py

```
@@ -0,0 +1,383 @@
+        refresh_db(sqlite_path)
+        try:
+            conn = sqlite3.connect(sqlite_path)
+            conn.row_factory = sqlite3.Row
+            lm_sentinel = has_sentinel(conn, 'lm_events')
+            olr_sentinel = has_sentinel(conn, 'olr_events')
+            lm_scn = get_max_scn(conn, 'lm_events')
+            olr_scn = get_max_scn(conn, 'olr_events')
+            lm_count = get_event_count(conn, 'lm_events')
+            olr_count = get_event_count(conn, 'olr_events')
+            conn.close()
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)